app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardKey.kt

```diff
@@ -106,13 +110,20 @@ fun KeyboardKey(
     onAutoCapitalize: (enable: Boolean) -> Unit,
     onSwitchLanguage: () -> Unit,
     onSwitchPosition: () -> Unit,
+    secondaryKey: KeyItemC? = null,
```

Comment:
There's something logically wrong here, because even if these are defined in the keyboard layout, you're overriding them below with `secondaryKeyboard = oppositeCase... secondaryKeyboard?.arr?.getOrNull(i)?.getOrNull(j)`

At least for now, rename these `oppositeCaseKey` and `numericKey` , and add a comment that they shouldn't be defined in the keyboard layout files.

Reply:
> As far as I know, keyboard layout files only configure KeyboardC, which contains the KeyItemCs for all keys, but they don't allow you to change all the fields in KeyboardKey, do they?

Ah that's correct. Ya just renaming them should be good.